### ISSUE TITLE
Add automated lookup of individuals VCFs in the output folder

### ISSUE THREAD

**Issue by nvnieuwk** [CenterForMedicalGeneticsGhent/nf-cmgg-germline#65]
### Description of feature

Add a feature that automatically looks in the `outdir` if the individuals VCFs were already generated by a previous run. Make this a toggle feature with a parameter which defaults to `false`

**Comment by nvnieuwk** [comment]
Is a precursor of #57

**Comment by matthdsm** [comment]
Not sure this will this work with S3 datastores

**Comment by nvnieuwk** [comment]
Yeah me neither, but I'll give it a try :) this would be very helpful when the pipeline results in a stupid HPC error

**Comment by nvnieuwk** [comment]
Also won't be a problem for the in house cluster, could still be useful to implement later but is not a priority
